src/_rules/background.js

```diff
@@ -37,13 +37,16 @@ export const backgrounds = [
   ['bg-origin-content', { 'background-origin': 'content-box' }],
   ...makeGlobalStaticRules('bg-origin', 'background-origin'),
 
-  //arbitrary
-  [/^bg-\[(.+)]/, ([, p]) => {
+  // matching arbitrary values
+  [/^bg-\[(.+)]/, ([, p, unit, context]) => {
```

Comment:
This does not look right? 🤔 It has only one matching group, so unit won't work... and context is supposed to go as the second parameter for the arrow function and not in the first array of matching groups. 🙂

Reply:
Oh you’re right! 🙈